hydra-plutus/src/Hydra/Contract/HeadState.hs

```diff
@@ -82,12 +82,12 @@ data CloseRedeemer
   = -- | Intial snapshot is used to close.
     CloseInitial
   | -- | Closing snapshot refers to the current state version
-    CloseCurrent
+    CloseUnused
       { signature :: [Signature]
       -- ^ Multi-signature of a snapshot ξ
       }
   | -- | Closing snapshot refers to the previous state version
-    CloseOutdated
+    CloseUsed
```

Comment:
Now I know why this changes the scripts:

https://github.com/input-output-hk/plutus/blob/bc8c3a765769d2c0cd41c43278f5954cfdfd9b15/plutus-tx/src/PlutusTx/IsData/TH.hs#L159-L161

Should: switch to `makeIsDataIndexed`

Reply:
why?

switching from
```hs 
PlutusTx.unstableMakeIsData ''CloseRedeemer
```
to
```hs
PlutusTx.makeIsDataIndexed ''CloseRedeemer [('CloseInitial, 0), ('CloseUnused, 1), ('CloseUsed, 3)]
```

still requires to change the scripts.

Reply:
Yes. But we would have not needed to change it if we used this before. Not a big deal, but I thought that if we change the scripts just for a rename, we could also do it better for the future (but we will change it anyways with inc commits)

Reply:
I prefer to introduce this suggestion as part of inc commits.

Appreciate for the catch!